### DISCUSSION TITLE
和暦 to 西暦

### DISCUSSION THREAD

**Discussion by zmcnulty** [melink14/rikaikun#1050] [Ideas]
Is stuff like this possible?
昭和58 -> 1983
昭和五十八年 -> 1983
大正元年 -> 1912

Looks like there is a different chrome extension that does this, but would be helpful to have it in rikaikun since I have it on a lot anyway!

**Comment by melink14** [comment]
Yes, I've seen this feature before but wasn't sure how much demand there was. It definitely seems useful! So I'll turn your post into a feature request!

**Reply by melink14** [reply]
https://github.com/melink14/rikaikun/issues/1052
